ui/app/templates/allocations/allocation/task/fs.hbs

```diff
@@ -1 +1,70 @@
-{{outlet}}
\ No newline at end of file
+{{task-subnav task=task}}
+<section class="section closer">
+  {{#if task.isRunning}}
+    {{! template-lint-disable table-groups }}
+    <table class="fs-explorer table is-compact">
+      <thead>
+        <tr>
+          <td colspan="3">
+            <nav class="breadcrumb fs" data-test-fs-breadcrumbs>
+              <ul>
+                <li class={{if breadcrumbs "" "is-active"}}>
+                  {{#link-to "allocations.allocation.task.fs-root" task.allocation task activeClass="is-active"}}
+                    {{task.name}}
+                  {{/link-to}}
+                </li>
+                {{#each breadcrumbs as |breadcrumb|}}
+                  <li class={{if breadcrumb.isLast "is-active"}}>
+                    {{#link-to "allocations.allocation.task.fs" task.allocation task breadcrumb.path activeClass="is-active"}}
+                      {{breadcrumb.name}}
+                    {{/link-to}}
+                  </li>
+                {{/each}}
+              </ul>
+            </nav>
+          </td>
+        </tr>
+      </thead>
+      {{#if isFile}}
+        <tbody>
+          <tr>
+            <td>
+              <div data-test-file-viewer>placeholder file viewer</div>
+            </td>
+          </tr>
+        </tbody>
+      {{else}}
+        {{#if directoryEntries}}
+          <thead>
+            <tr>
+              <th class="is-two-thirds">Name</th>
+              <th class="has-text-right">File Size</th>
+              <th class="has-text-right">Last Modified</th>
```

Comment:
Not necessary to do in this PR, but I don't think it will take long before we get requests to support sorting here. Sorting is pretty standardized throughout Nomad and well-factored, so it's worth exploring as part of this feature branch if not this PR.

Reply:
indeed, I wondered about that. I’ll make a separate PR.